packages/lib-grommet-theme/src/index.js

```diff
@@ -425,7 +425,8 @@ const theme = deepFreeze({
   },
   radioButton: {
     icon: {
-      size: 'small'
+      size: '15px',
+      extend: props => `circle { r: 10px; }`
```

Comment:
Not a blocker, but this doesn't need to be a function, as it's not using `props` - it can just be:

```
extend: 'circle { r: 10px; }'
```